bundles/org.openhab.binding.androidtv/src/main/resources/OH-INF/i18n/androidtv.properties

```diff
@@ -84,5 +86,16 @@ offline.interrupted = Interrupted
 offline.io-error = I/O Error
 offline.runtime-exception = Runtime exception
 offline.user-forced-pin-process = User Forced PIN Process
+offline.error-occured-while-presenting-pairing-code = Error occurred while trying to present a Pairing Code on TV
+offline.error-occured-during-retrieval-of-credentials = Error occurred during retrieval of credentials
+offline.pairing-is-not-configured-yet = Pairing is not configured yet
+offline.error-occurred-while-trying-to-present-a-pairing-code-on-tv = Error occurred while trying to present a Pairing Code on TV
+offline.pairing-code-is-available-but-credentials-missing = Pairing Code is available, but credentials missing. Trying to retrieve them
+offline.error-occurred-during-retrieval-of-credentials = Error occurred during retrieval of credentials
+offline.pairing-was-unsuccessful = Pairing was unsuccessful
+offline.error-occurred-during-creation-of-http-client = Error occurred during creation of HTTP client
+offline.authentication-with-philips-tv-device-was-successful-continuing-initialization-of-the-tv = Authentication with Philips TV device was successful. Continuing initialization of the tv.
+offline.could-not-successfully-finish-pairing-process-with-the-tv = Could not successfully finish pairing process with the TV
```

Comment:
It seems that these are prematurely introduced, would be better to revert and only introduce with the PR using them.